docusaurus.config.js

```diff
@@ -146,16 +146,7 @@ module.exports = {
           title: "Community",
           items: [
             {
-              label: "Slack",
-              href: "https://camunda.com/slack",
-            },
-            {
-              label: "Twitter",
-              href: "https://twitter.com/camunda",
-            },
-            {
-              label: "GitHub",
-              href: "https://github.com/camunda/camunda-platform-docs",
+              html: `<a href="https://camunda.com/slack" target="_blank" rel="noreferrer noopener"><img src= "img/Slack-mark-white-RGB.png" class="footer-logos" /></a> <a href="https://twitter.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/twitter.svg" class="footer-logos" /></a> <a href="https://github.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/github-mark-white.svg" class="footer-logos" /></a>`,
```

Comment:
```suggestion
              html: `<a href="https://camunda.com/slack" target="_blank" rel="noreferrer noopener"><img src= "img/Slack-mark-white-RGB.png" alt="Camunda on Slack" class="footer-logos" /></a> <a href="https://twitter.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/twitter.svg" alt="Camunda on Twitter" class="footer-logos" /></a> <a href="https://github.com/camunda" target="_blank" rel="noreferrer noopener"><img src= "img/github-mark-white.svg" alt="Camunda on GitHub" class="footer-logos" /></a>`,
```

Following [guidance from the w3c](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo), we should add `alt` attributes to the images for assistive devices. Feel free to change those suggested `alt` attributes if you don't like them 😃

Reply:
THANK YOU! Will add.